src/main/java/com/boot/bankservice/service/AccountService.java

```diff
@@ -0,0 +1,16 @@
+package com.boot.bankservice.service;
+
+import com.boot.bankservice.model.Account;
+import java.util.Optional;
+
+public interface AccountService {
+    Account add(Account account);
+
+    Optional<Account> get(Long id);
```

Comment:
let's return entity from the service layer

and also add the method to get all accounts of user